documentation/src/docs/rest/restapis.md

```diff
@@ -417,6 +417,12 @@ Create a new stream
       "days" : 0,
       "hours" : 0,
       "minutes" : 0
+    },
+    "maxValue" : 0,
+    "maxTimeBasedRetention" : {
+      "days" : 0,
+      "hours" : 0,
+      "minutes" : 0
     }
   }
 }
```

Comment:
A couple of questions: 
- HTTP response codes for the auth failures should be added too.
- `StreamCOnfiguration.timestampAggregationTimeout` is not present as part of the StreamConfiguration for REST APIs. Should this be added ?

Reply:
yes, in fact the rest also needs the new values for CBR that were added which is not present. @pbelgundi could you please fix bot of these. thanks